tensorflow/lite/micro/kernels/svdf.h

```diff
@@ -35,6 +35,17 @@ struct OpData {
   int output_zero_point;
 };
 
+// Input tensors.
+constexpr int kInputTensor = 0;
+constexpr int kWeightsFeatureTensor = 1;
+constexpr int kWeightsTimeTensor = 2;
+constexpr int kBiasTensor = 3;
+// This is a variable tensor, and will be modified by this op.
+constexpr int kInputActivationStateTensor = 4;
+
+// Output tensor.
+constexpr int kOutputTensor = 0;
```

Comment:
make extern const and add Svdf prefix (e.g. kSvdfInputTensor)

https://github.com/tensorflow/tensorflow/blob/d73ba75a4c1b26b91cc20cb86866d775c02dc479/tensorflow/lite/micro/kernels/fully_connected.h#L43-L46


See section on constants in header files: https://abseil.io/tips/140

Since we are not using c++17, we can not use inline constexpr and and extern const is the way to go for us.

And add definition in svdf_common.cc:
https://github.com/tensorflow/tensorflow/blob/d73ba75a4c1b26b91cc20cb86866d775c02dc479/tensorflow/lite/micro/kernels/fully_connected_common.cc#L29-L32

Reply:
Done